pkg/restore/crd.go

```diff
@@ -42,7 +43,16 @@ func RestoreCrd(ctx context.Context, k8sclientset *k8sutil.Clientsets, operatorN
 	}
 
 	logging.Info("Detecting which resources to restore for crd %q", crd)
-	crdList, err := k8sclientset.ListResourcesDynamically(ctx, crds.CephRookIoGroup, crds.CephRookResourcesVersion, crd, clusterNamespace)
+	var groupName, versionResource string
+	// Set default values for groupName and versionResource
+	if groupNameOptional == nil || versionResourceOptional == nil {
```

Comment:
No need for these checks if we just require them

Reply:
done